<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetAES</class>
 <widget class="QWidget" name="WidgetAES">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1103</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTextEdit" name="textEdit"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_algorithm">
       <property name="text">
        <string>加密算法</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_algorithm"/>
     </item>
     <item>
      <widget class="QLabel" name="label_mode">
       <property name="text">
        <string>算法模式</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_mode"/>
     </item>
     <item>
      <widget class="QLabel" name="label_padding">
       <property name="text">
        <string>填充</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_padding"/>
     </item>
     <item>
      <widget class="QLabel" name="label_block_length">
       <property name="text">
        <string>数据块</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_block_length"/>
     </item>
     <item>
      <widget class="QLabel" name="label_password">
       <property name="text">
        <string>密码</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_password"/>
     </item>
     <item>
      <widget class="QLabel" name="label_iv_offset">
       <property name="text">
        <string>偏移量</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_iv_offset"/>
     </item>
     <item>
      <widget class="QLabel" name="label_output">
       <property name="text">
        <string>输出</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_output"/>
     </item>
     <item>
      <widget class="QLabel" name="label_char_set">
       <property name="text">
        <string>字符集</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_charset"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextBrowser" name="textBrowser"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_encrypt">
       <property name="text">
        <string>加密</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_decrypt">
       <property name="text">
        <string>解密</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
